internal/configs/configurator.go

```diff
@@ -59,8 +59,11 @@ const JWTKeyKey = "jwk"
 // HtpasswdFileKey is the key of the data field of a Secret where the HTTP basic authorization list must be stored
 const HtpasswdFileKey = "htpasswd"
 
-// CAKey is the key of the data field of a Secret where the cert must be stored.
-const CAKey = "ca.crt"
+// CACrtKey is the key of the data field of a Secret where the cert must be stored.
+const CACrtKey = "ca.crt"
+
+// CACrlKey is the key of the data field of a Secret where the cert revocation list much be stored.
```

Comment:
```suggestion
// CACrlKey is the key of the data field of a Secret where the cert revocation list must be stored.
```

May also benefit from clarifying this field as optional?